src/js/utils.mjs

```diff
@@ -71,8 +71,8 @@ function loadTemplate(path) {
 } 
 
 export async function loadHeaderFooter() {
-  const headerTemplateFn = loadTemplate("/partials/header.html");
-  const footerTemplateFn = loadTemplate("/partials/footer.html");
+  const headerTemplateFn = loadTemplate("./public/partials/header.html");
+  const footerTemplateFn = loadTemplate("./public/partials/footer.html");
```

Comment:
@bloominolive These should be reverted to just be `/partials/header.html` and `/partials/footer.html`. The reason is that the build process just copies everything from the public directory into the dist directory. The dist directory is where everything is served from on netlify. So in this case the problem was that the partials were never making it to the dist directory so `/partials` wasn't available.